platform-service-framework/src/main/java/org/hypertrace/core/serviceframework/config/ConfigUtils.java

```diff
@@ -3,26 +3,40 @@
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigRenderOptions;
 import com.typesafe.config.ConfigValue;
+import com.typesafe.config.ConfigValueType;
 import java.io.PrintWriter;
 import java.io.StringWriter;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
 import java.util.Properties;
+import java.util.Set;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class ConfigUtils {
 
-  private static Logger LOGGER = LoggerFactory.getLogger(ConfigUtils.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(ConfigUtils.class);
+  private static final Set<String> SECRET_KEYS = Set.of("password", "secret");
```

Comment:
So I certainly don't want to log passwords and secrets, but I'd question why we're putting them in config files and logging them in the first place? Feel free to ping me if sensitive.

On the actual impl, rather than hardcoding these can we overload logConfFile with a method that takes keys to redact - you can route the single arg version that exists to use these two as the defaults, but that leaves consumers in control without having to modify the framework.